.evergreen/.evg.yml

```
@@ -111,6 +111,9 @@ functions:
           ${PROJECT_DIRECTORY}/${file}
 
   "upload mo artifacts":
+    - command: ec2.assume_role
```

It should be a function.

```suggestion
functions:
  assume-ec2-role:
    command: ec2.assume_role
    params:
      role_arn: ${ASSUME_ROLE_ARN}
```

---

And every task that requires iam permissions should call this function

---

`upload mo artifacts` is a function, and I believe functions cannot call other functions. This seems to be the only function that uses `aws_key`, so I believe everything should work correctly.

---

Discussed offline, we're fine to keep as is.